crates/sui-types/src/multisig.rs

```diff
@@ -305,3 +319,30 @@ impl MultiSigPublicKey {
         Ok(())
     }
 }
+
+impl ToFromBytes for MultiSig {
+    fn from_bytes(bytes: &[u8]) -> Result<MultiSig, FastCryptoError> {
+        // The first byte matches the flag of MultiSig.
+        if bytes[0] != SignatureScheme::MultiSig.flag() {
```

Comment:
will panic if bytes = [] (or below if bytes is of length 1)
let's first verify the length of bytes